packages/tangle/tangle.go

```diff
@@ -71,7 +72,10 @@ func (t *Tangle) Setup() {
 	t.Requester.Setup()
 	t.TipManager.Setup()
 	t.Scheduler.Setup()
-	t.OpinionFormer.Setup()
+	if !t.Options.WithoutOpinionFormer {
+		t.OpinionFormer.Setup()
+		return
+	}
```

Comment:
I would leave comment here: "Booker and LederState setup is left out until the old value tangle is in use."